nautobot_ssot/integrations/aci/jobs.py

```diff
@@ -71,19 +64,45 @@ def data_mappings(cls):
 
     def load_source_adapter(self):
         """Method to instantiate and load the ACI adapter into `self.source_adapter`."""
-        self.source_adapter = AciAdapter(job=self, sync=self.sync, client=aci_creds[self.apic])
+        if not self.device_site:
+            self.logger.info("Device Location is unspecified so will revert to specified Controller's Location.")
```

Comment:
Wondering if we should use `controller.location` or `controller.controller_device.location` - any thoughts ?

Reply:
I think when I discussed this previously we agreed upon defaulting to the Location on the Controller device itself.